docs/site/sidebars/lb4_sidebar.yml

```diff
@@ -364,15 +364,43 @@ children:
           output: 'web, pdf'
 
     - title: 'Authorization'
-      url: Loopback-component-authorization.html
+      url: Authorization-overview.html
       output: 'web, pdf'
+      children:
+        - title: 'RBAC with Authorization System'
```

Comment:
Move it to be after `Authorization Component`?

Reply:
Thanks @raymondfeng I put as the first tutorial on purpose, like we have jwt-authentication tutorial for authentication. 
Here is how the tutorials are arranged:
People learn with building a RBAC system to get familiar with the concepts. 
Then they learn the details of using the component.